cmd/genutils/genutils.go

```diff
@@ -39,3 +41,65 @@ func OutDir(path string) (string, error) {
 	outDir = outDir + "/"
 	return outDir, nil
 }
+
+func getOutDir(path string, args []string) string {
+	if len(args) == 1 {
+		path = args[0]
+	} else if len(args) > 1 {
+		fmt.Fprintf(os.Stderr, "usage: COMMAND [output directory]\n")
+		os.Exit(1)
```

Comment:
Harsh error case for such an innocuous method name.  Return an error and the keep the exits/fatals in the top level of the command.